### PR TITLE
Adjust CI configuration

### PR DESCRIPTION
**Pull request by kvch**
## What does this PR do?

This PR adjust the path of the generators as these have been moved.

## Why is it important?

It has to be adjusted, otherwise the CI jobs cannot start.

## Checklist

~~- [ ] My code follows the style guidelines of this project~~
~~- [ ] I have commented my code, particularly in hard-to-understand areas~~
~~- [ ] I have made corresponding changes to the documentation~~
~~- [ ] I have made corresponding change to the default configuration files~~
~~- [ ] I have added tests that prove my fix is effective or that my feature works~~

Requies #16127

### CODE REVIEW

File: dev-tools/jenkins_ci.ps1 [hunk 1]
```diff
@@ -32,7 +32,7 @@ $env:TEST_COVERAGE = "true"
 $env:RACE_DETECTOR = "true"
 
 # Install mage from vendor.
-exec { go install github.com/elastic/beats/vendor/github.com/magefile/mage } "mage install FAILURE"
+exec { go install -mod=vendor github.com/magefile/mage } "mage install FAILURE"
```
Comment (ycombinator) [inline dev-tools/jenkins_ci.ps1]: Just for my understanding, this change is not really related to the `_template` path change, right? It's just another change needed when moving to go modules?

**Comment by kvch** [comment]
jenkins test this
